File: dotNet/test/Wall/DummyWallDAO.cs
using System.Collections.Immutable;
using src.Wall;

namespace test.Wall;

public class DummyWallDAO : IWallDAO
{
    public ImmutableArray<Brick> GetBricks(src.User.User user)
    {
        throw new NotImplementedException();
    }

    public void AddBrick(src.User.User user, Brick brick)
    {
        throw new NotImplementedException();
    }
}

File: typescript/test/Wall/DummyWallDAO.ts
import {WallDAOInterface} from '../../src/Wall/WallDAOInterface';
import {User} from '../../src/User/User';
import {Brick} from '../../src/Wall/Brick';

export class DummyWallDAO implements WallDAOInterface {

    getBricks(user: User): ReadonlyArray<Brick> {
        throw new Error("not implemented")
    }

    addBrick(user: User, brick: Brick): void {
        throw new Error("not implemented")
    }
}
